templates/templates/footer.html

```diff
@@ -219,6 +219,16 @@ <h2 class="p-footer__title p-footer__title-text">
                 <path class="cls-2" d="M7 8.512v16.38c0 .758.63 1.37 1.404 1.37h16.192c.775 0 1.404-.612 1.404-1.37V8.512c0-.755-.63-1.37-1.404-1.37H8.404C7.63 7.143 7 7.757 7 8.513zm5.76 14.636H9.89v-8.634h2.87v8.634zm-1.435-9.812h-.02c-.962 0-1.585-.663-1.585-1.492 0-.847.642-1.492 1.624-1.492s1.586.645 1.604 1.492c0 .83-.623 1.492-1.623 1.492zm3.022 9.812s.038-7.824 0-8.634h2.87v1.252h-.02c.38-.59 1.058-1.454 2.607-1.454 1.888 0 3.303 1.234 3.303 3.885v4.95h-2.87V18.53c0-1.162-.415-1.953-1.453-1.953-.793 0-1.265.534-1.472 1.05-.076.184-.095.44-.095.7v4.82h-2.87z" />
               </svg></a>
             </li>
+            <li class="p-inline-list__item">
+              <a class="p-inline-list__link--twitter" title="Follow Ubuntu on Instagram" href="https://www.instagram.com/ubuntu_os/">
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 375 375">
```

Comment:
I think this is currently missing the styles for the bg and the actual icon - compare with how it is done for the other icons. I think adding the styles as suggested below should solve this, but it is just an assumption, so please verify the result on your end :)
```suggestion
                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 375 375">
                  <defs>
                   <style>
                     .instagram-icon {
                       fill: #666666;
                     }
                     .cls-2 {
                       fill: #fff;
                     }
                   </style>
                 </defs>
```

Then these classes need to be applied to the 2 `path` elements, one for the bg and the other for the actual icon - see suggestions below.